res/controllers/Korg-KAOSS-DJ-scripts.js

```diff
@@ -137,68 +143,168 @@ KAOSSDJ.wheelTurnShift = function(channel, control, value, status, group) {
         engine.scratchTick(deck.deckNumber, newValue);
     } else {
         // Move beatgrid
-        if(value === UP){
-            engine.setValue(deck.group, 'beats_translate_later', true);
-        } else if (value === DOWN){
-            engine.setValue(deck.group, 'beats_translate_earlier', true);
+        if (value === UP) {
+            engine.setValue(deck.group, "beats_translate_later", true);
+        } else if (value === DOWN) {
+            engine.setValue(deck.group, "beats_translate_earlier", true);
         }
     }
 };
 
-KAOSSDJ.scratchMode = function(channel, control, value, status, group) {
-    if (value === ON) {
-        // Turn scratch mode on jogwheel (LED is red)
-        KAOSSDJ.updateDeckByChannel(channel, 'jogWheelsInScratchMode', true);
-    } else if (value === OFF) {
-        // Turn scratch mode on jogwheel (LED is off)
-        KAOSSDJ.updateDeckByChannel(channel, 'jogWheelsInScratchMode', false);
-    }
+KAOSSDJ.scratchMode = function(channel, _control, value, _status, _group) {
+    KAOSSDJ.updateDeckByChannel(channel, "jogWheelsInScratchMode", value === ON);
 };
 
-KAOSSDJ.browseKnob = function(channel, control, value, status, group) {
-    var nval = (value > 0x40 ? value - 0x80 : value);
-    engine.setValue(group, "SelectTrackKnob", nval);
+KAOSSDJ.fxToggleButton = function(channel, _control, value, _status, _group) {
+    KAOSSDJ.updateDeckByChannel(channel, "fx", value === ON);
 };
 
-KAOSSDJ.leftFxSwitch = function(channel, control, value, status, group) {
-    var deck = KAOSSDJ.getDeckByChannel(channel);
-    if(value === ON) {
-        KAOSSDJ.updateDeckByChannel(channel, 'fx', true);
-    } else {
-        KAOSSDJ.updateDeckByChannel(channel, 'fx', false);
-    }
+KAOSSDJ.fxKnob = function(_channel, _control, _value, _status) {
+    // TODO
 };
```

Comment:
do you want to finish this before merge?

Reply:
Not yet. It is still unclear how to best map it in Mixxx.